tests/hazmat/primitives/test_smime.py

```diff
@@ -0,0 +1,395 @@
+# This file is dual licensed under the terms of the Apache License, Version
+# 2.0, and the BSD License. See the LICENSE file in the root of this repository
+# for complete details.
+
+from __future__ import absolute_import, division, print_function
```

Comment:
None of these are necessary in any currently supported python version.

Reply:
We support Python 2.7 still, and these are absolutely necessary to ensure consistent behavior.

Reply:
Python 2.7 has been unsupported upstream since the beginning of 2020.
It's not even subject to security maintenance. It probably makes sense to drop it, right?

Reply:
If you're interested in our Python 2.7 plans, you can follow along here: https://github.com/pyca/cryptography/issues/5359

Commenting on completely unrelated PRs about it is not productive.